hcfb-fe/src/app/distribution/log/log.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="d-flex justify-content-center" style="margin-top: 3em; margin-bottom: 2em;">
+   <button routerLink="/main" type="button" class="btn btn-nav"><i class="fas fa-arrow-left" style="font-size: 2em">
+   </i></button>
+   <h1>Daily Distribution</h1>
+   <button type="button" class="btn btn-nav"><i class="fas fa-bars" style="font-size: 2em">
+   </i></button>
+</div>
+<div class="d-flex justify-content-center">
+   <div class="weight-font" style="min-width: 25em">The Connection Church 
+   </div>
+</div>
+
+<div class="d-flex justify-content-right">
+   <div class="weight-font" style="min-width: 20em">Lbs
+   </div>
+</div>
+
+<div class="d-flex justify-content-center">
+   <div class="weight-font" style="min-width: 25em">Breads</div>
+   <input type="number" name="email" size="4"> <button type="button" class="btn"><i class="fas fa-plus-circle add"></i></button> 
```

Comment:
I'm not too sure why the name is email for this input. It's definitely not a big deal but you might want to change that on a later commit.